src/components/App/index.tsx

```diff
@@ -70,58 +44,37 @@ const App: React.FC = ({ children }) => {
 
   return (
     <Frame>
-      <SnackbarProvider
```

Comment:
To be able to test that the Snackbar is showed, I moved our `<SnackbarProvider />` component from `src/components/App/index.tsx` to `/src/components/Providers/index.tsx` (where all our providers are defined)